### PR TITLE
Add object for displaying newly added comment

### PR DESCRIPTION
**Pull request by AbeeraTahir**
In this PR, I have added a JavaScript object to store the user comment with its details. I have used this object for displaying the newly added comment to the comment list so that we don't need to call the API again for fetching all the comments to update the screen.

### CODE REVIEW

File: src/modules/render.js [hunk 1]
```diff
@@ -58,7 +55,11 @@ const showPopup = async (ebook) => {
   commentsArray.sort((a, b) => new Date(b.creation_date) - new Date(a.creation_date));
 
   // displaying comments
-  const allComments = `<div class="comment-list">${showComments(commentsArray)}</div>`;
+  let allComments = '<div class="comment-list">';
+  commentsArray.forEach((x) => {
+    allComments += showComments(x);
+  });
```
Comment (lmunozm1702) [inline src/modules/render.js]: We can improve ES6 by writing this forEach in one line only. And change the "x" named variable to another more meaningful one.

**Review by lmunozm1702 — CHANGES_REQUESTED** [review]
Hi @AbeeraTahir, 
 Nice work refactoring to use classes, just a couple of comments in the code to improve ES6.

dog-so-close


Happy Coding.

File: src/modules/render.js (hunk 1 above)
Comment (AbeeraTahir) [reply src/modules/render.js]: Hi @lmunozm1702, writing function into one line is throwing linter error "Arrow function should not return assignment  no-return-assign". I just went ahead with renaming the variable.

**Review by lmunozm1702 — APPROVED** [review]
Great work, the code look's amazing!!!

approved

You can do the merge now!

**Pull request merged**